.github/workflows/check-features.yml

```diff
@@ -15,7 +15,7 @@ concurrency:
 jobs:
   check:
     name: Check workspace features
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
What is the motivation ? 

A new ubuntu version could create problems and break CI in the future. Not sure if this is a good idea as it makes the environment less stable.

Reply:
> What is the motivation ?
> 
> A new ubuntu version could create problems and break CI in the future. Not sure if this is a good idea as it makes the environment less stable.

Hmm, good point. Actually, I was just trying to align it because some workflows use `ubuntu-22.04`, while others use `ubuntu-latest`.  

I checked `polkadot-sdk`, and it also uses `runs-on: ubuntu-latest` everywhere, except for some custom runners like `parity-large` (maybe also use `ubuntu-latest`) and `parity-macos`.  

I'm not sure - I can revert the related changes and create a separate issue for this, but that would mean having two different versions in the CI (`ubuntu-22.04` and `ubuntu-latest`) anyway.

@sandreim wdyt?

Reply:
> Can't we use a fixed version everywhere ?

@sandreim Reverted here and created separate PR: https://github.com/polkadot-fellows/runtimes/pull/645